config.json

```diff
@@ -27,6 +27,8 @@
     "comm-central": {
       "index_path": "$WORKING/comm-central",
       "files_path": "$WORKING/comm-central/comm-central",
+      "git_path": "$WORKING/comm-central/comm-central",
+      "git_blame_path": "$WORKING/comm-central/comm-central-blame",
       "hg_root": "https://hg.mozilla.org/comm-central",
```

Comment:
Hmm, does this need to go away?

Reply:
hg_root is still used for "raw" links, and to get icons in the directory listing at https://github.com/mozsearch/mozsearch/blob/07726ffa17fbc9fc4ec116603bdcbea7b31d7aa6/output-dir.js#L99 for example. So it still provides some value.